Xamarin.Forms.Platform.WinRT/LabelRenderer.cs

```diff
@@ -172,6 +204,7 @@ void UpdateLineBreakMode(TextBlock textBlock)
 					textBlock.TextWrapping = TextWrapping.NoWrap;
 					break;
 				case LineBreakMode.MiddleTruncation:
+					// TODO: This truncates at the end.
```

Comment:
Can we add some more detail on this TODO? Is the noted behavior something we can fix, or is it just that WinRT doesn't natively support this type of truncation?